src/ucp/api/ucp.h

```diff
@@ -1137,6 +1150,25 @@ struct ucp_tag_recv_info {
 };
 
 
+/**
+ * @ingroup UCP_CONTEXT
```

Comment:
`UCP_WORKER` seems to be a better match. Also `UCP_WORKER` group is stated for `ucp_tag_recv_cancel_params_field`